src/pages/planters/[planterid].js

```diff
@@ -97,6 +97,8 @@ export default function Planter(props) {
 
   const { classes } = useStyles();
 
+  console.log('planter', planter);
```

Comment:
loglevel is better than console, we can control the level (warn, info) to print at the starting point of the app, and change it on the fly (in browser)

Reply:
I'm going to have to look that up, I try to remember to delete all the logs that I put in before I'm done.